src/SubscriptionContextFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\SubscriptionContext;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use Doctrine\Common\Persistence\Mapping\Driver\MappingDriver;
+use Doctrine\ORM\Mapping\Driver\XmlDriver;
+use Gedmo\Timestampable\TimestampableListener;
+
+class SubscriptionContextFactory {
+
+	private const DOCTRINE_CLASS_MAPPING_DIRECTORY = __DIR__ . '/../config/DoctrineClassMapping';
+
```

Comment:
We still need the public constant `ENTITY_NAMESPACE` for the chained mapping driver in the FF